build/docs-manager/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN apk add python3-dev
 RUN apk add libc6-compat
 RUN apk add --no-cache --virtual .deps gcc musl-dev
 RUN apk add --no-cache git bash openssh ruby && \
-    gem install mdl
```

Comment:
I think you could set `golang-build` to `v1.3.0` and `FROM alpine:3.16` while you're at it